MapboxSearch/autofill/src/androidTest/java/com/mapbox/search/autofill/AddressAutofillIntegrationTest.kt

```diff
@@ -78,7 +74,6 @@ internal class AddressAutofillIntegrationTest {
 
         val url = requireNotNull(request.requestUrl)
         assertEqualsIgnoreCase("//autofill/v1/suggest/${query.query}", url.encodedPath)
-        assertEquals(TEST_ACCESS_TOKEN, url.queryParameter("access_token"))
```

Comment:
Token still should be passed to backend. This check should remain here. The same for other checks in this file

Reply:
I rolled back that change. Fixed.